src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class Car implements Comparable<Car> {
+    private static final int MIN_VALUE = 0;
+    private static final int MAX_VALUE = 9;
+    private static final int ADVANCE_VALUE = 4;
+    private final String name;
+    private int move = 0;
+
+    public Car(final String name) {
+        this.name = name;
+    }
+
+    @Override
+    public int compareTo(Car o) {
+        return o.move - this.move;
+    }
+
+    public boolean isWinner(final int maxMove) {
+        if (move >= maxMove)
+            return true;
+
+        return false;
+    }
```

Comment:
조건문을 그대로 반환해주면 간단할 것 같아요!😃
```suggestion
    public boolean isWinner(final int maxMove) {
        return move >= maxMove;
    }
```

Reply:
헉 그러네요! 놓친 부분 알려주셔서 감사합니다👍🏻